.github/workflows/delivery.yml

```diff
@@ -1,16 +0,0 @@
-name: delivery
```

Comment:
Why is this being removed?

Reply:
@btm suggested that delivery can be removed as it no longer exists.

Reply:
This was a delivery local mode config which is still used by chef projects

Reply:
Thanks for the review @tas50 , i will revert this commit.